R/manualRecode.R

```diff
@@ -0,0 +1,14 @@
+#' Get manual recodes
+#'
+#' @param recodedList A data frame with the automatically recoded values.
+#' @return Returns a data frame wich only includes the values that have to be recoded manually. This can be saved to excel for manual recoding.
+#'
+#' @examples # tbd
+
+manualRecode <- function(recodedList){
```

Comment:
I prefer verbs for function names. How about `extractManualRecodes()`? Of course it's a bit long and clunky, but similar to the other function names?